### ISSUE TITLE
Is there any pre-trained model?like pretrained on ImageNet.Because I have trained CIFAR100 from the head,but the accuarcy only reaches 60%

### ISSUE THREAD

**Issue by jancylee** [hysts/pytorch_image_classification#7]

**Comment by hysts** [comment]
Hi, @jancylee 
Sorry for the late reply.

No, you don't need any pretrained model to initialize weights. Which model did you use? Because CIFAR-100 is more difficult to train than CIFAR-10, the performance of the model may simply be the value.

**Comment by hysts** [comment]
<img width="776" alt="1603 05027_table3" src="https://user-images.githubusercontent.com/25161192/78988490-06439c00-7b6c-11ea-9ad2-523bf077fefb.png">

According to the [ResNet (with preactivation)](https://arxiv.org/abs/1603.05027) paper, the accuracy of ResNet-164 is around 75%. So if your model is smaller one than it, 60% accuracy is understandable. If not, the older version of this repository could have had some bugs.
FYI, with the latest head, the accuracy of ResNet-164 trained on CIFAR-100 is 73.4% (not an average of multiple runs). It's slightly lower than expected, and I find it a little bit worrisome, but it could be due to noise or differences in hyperparameters.
